diff --git a/CMakeLists.txt b/CMakeLists.txt
index df0054e7d9..ce870ba6bc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -268,6 +268,11 @@ include(cmake/ssl.cmake)
 include(cmake/upb.cmake)
 include(cmake/zlib.cmake)
 
+if (ABSL_USE_CXX17)
+    message(STATUS "Found absl uses CXX17, enable CXX17 feature.")
+    set(CMAKE_CXX_STANDARD 17)
+endif()
+
 if(_gRPC_PLATFORM_MAC OR _gRPC_PLATFORM_IOS)
   set(_gRPC_ALLTARGETS_LIBRARIES ${CMAKE_DL_LIBS} m pthread)
 elseif(_gRPC_PLATFORM_ANDROID)
