diff --git a/util/pl/VC-32.pl b/util/pl/VC-32.pl
index dba96cb..87ccd59 100644
--- a/util/pl/VC-32.pl
+++ b/util/pl/VC-32.pl
@@ -130,7 +130,7 @@ elsif ($FLAVOR =~ /CE/)
 else	# Win32
     {
     $base_cflags= " $mf_cflag";
-    my $f = $shlib || $fips ?' /MD':' /MT';
+    my $f = $shlib || $fips ?' /MT':' /MT';
     $ff = "/fixed";
     $opt_cflags=$f.' /Ox /O2 /Ob2';
     $dbg_cflags=$f.'d /Od -DDEBUG -D_DEBUG';
